packages/ERTP/src/paymentLedger.js

```diff
@@ -16,12 +16,13 @@ import '@agoric/store/exported.js';
  * Make the paymentLedger, the source of truth for the balances of
  * payments. All minting and transfer authority originates here.
  *
+ * @template {AssetKind} [K=AssetKind]
  * @param {string} allegedName
  * @param {Brand} brand
  * @param {AssetKind} assetKind
```

Comment:
Again, `brand` is how the caller picks `K`, right?

```suggestion
 * @param {Brand<K>} brand
 * @param {AssetKind} assetKind
```

The `assetKind` should match `K` too, but I don't know whether there's a straightforward way to express that.